externs/browser/url.js

```diff
@@ -22,10 +22,20 @@
  * @author rdcronin@google.com (Devlin Cronin)
  */
 
+/**
+ * @typedef {IArrayLike<string>}
+ */
```

Comment:
Please add this descriptive comment or a more correct version.

`* In practice this is an array with 2 strings. The first is a parameter name. The second is its value.`

Reply:
obsoleted by my comment above.